apps/src/templates/census2017/CensusForm.jsx

```diff
@@ -300,7 +314,7 @@ class CensusForm extends Component {
                 )}
               </div>
               <select
-                name="hoc_s"
+                name="how_many_do_hoc"
```

Comment:
So we don't need the data type `_s` appended to the name because that's an expectation of Pegasus forms, but not of your new table?

Reply:
Yes. I changed the naming in the forms to match the models and controller rather than making the models and controller match the existing form fields.

Reply:
The `_s` suffixes are for SOLR to understand the data types when we index Pegasus forms. We don't use SOLR on the dashboard side, so don't need it.